BizHawk.Client.Common/movie/tasproj/TasStateManager.cs

```diff
@@ -404,26 +404,6 @@ public int GetStateFrameByIndex(int index)
 		}
 
 		private ulong _used;
-		private ulong Used
-		{
-			get
-			{
-				return _used;
-			}
-
-			set
-			{
-				// TODO: Shouldn't we throw an exception? Debug.Fail only runs in debug mode?
-				if (value > 0xf000000000000000)

```

Comment:
It was a problem when branch states were changing `Used` all the time while being terrible at it. Maybe not needed anymore.

Reply:
Yes, pretty sure this is no longer needed